infra/core/env/managed-cluster/.bicep/prometheus-rule-groups.bicep

```diff
@@ -0,0 +1,181 @@
+@description('Required. Azure Monitor Workspace resource id.')
+param monitorWorkspaceResourceId string 
+@description('Required. The clustername to scope the data collection rule association.')
+param clusterName string
+@description('Required. The AKS Cluster resource is.')
+param clusterResourceId string
+@description('Required. The Azure region where the resources will be deployed.')
+param location string
+
+var nodeRecordingRuleGroup = 'NodeRecordingRulesRuleGroup-'
+var nodeRecordingRuleGroupName = '${nodeRecordingRuleGroup}${clusterName}'
+var nodeRecordingRuleGroupDescription = 'Node Recording Rules RuleGroup'
+var kubernetesRecordingRuleGrouPrefix = 'KubernetesReccordingRulesRuleGroup-'
+var kubernetesRecordingRuleGroupName = '${kubernetesRecordingRuleGrouPrefix}${clusterName}'
+var kubernetesRecordingRuleGroupDescription = 'Kubernetes Recording Rules RuleGroup'
+var version = ' - 0.1'
+
+resource nodeRecordingRuleGroupResource 'Microsoft.AlertsManagement/prometheusRuleGroups@2023-03-01' = {
```

Comment:
We should document the alert rules on the wiki in the monitoring and alerting section.

Reply:
These are the default metrics required to populate the Grafana Dashboard and not the alert rules.